.gitpod.yml

```diff
@@ -1,19 +1,29 @@
-image:
-  file: testing/tools/gitpod/Dockerfile
+image: quansight/jupyter-a11y:latest
 
 tasks:
   - name: Install JupyterLab Node.js testing dependencies
     init: |
       cd testing/jupyterlab
+      echo " 📦 Installing node dependencies "
       yarn install
-      cd workspace/a11y
+      npx playwright install
+      echo "🚀 Dependencies installed "
+      cd /workspace/accessibility/
```

Comment:
Not sure 🤔  I would prefer adding this to the docs instead of prodding them to run tests straight away

Reply:
That makes sense. I'm thinking about how there are maybe two ways that I can think that somebody might end up in the Gitpod workspace:

1. They start with the readme in testing/jupyterlab
2. They see the .gitpod.yml file and/or they guess that they can get into Gitpod using gitpod.io

The user who takes the first path should be fine. The user who takes the second path might be a little lost. And if you take the second path, it's a little difficult to even know where to go to figure out what to do because the relevant readme isn't at the top-level, it's at accessibility/testing.